cmd/scheduled-task-adder/main.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"cmp"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/google/uuid"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/donor/donordata"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/dynamo"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/identity"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/scheduled"
+)
+
+var taskCount int
+var entryPoint string
+
+type TaskCountEvent struct {
+	TaskCount int `json:"taskCount"`
+}
+
+func init() {
+	flag.IntVar(&taskCount, "taskCount", 0, "Number of scheduled tasks to generate")
```

Comment:
I'd originally made this as a local script to run so have kept the option of building and running it outside of docker. I'm not sure how much use it will get in this form so if we think YAGNI I can simplify to be just a lambda.